operations/firmware/FAS_CLI.md

```diff
@@ -696,36 +700,43 @@ loaderRunID = "8efb19c4-77a2-41da-9a8f-fccbfe06f674"
 ```
 The loader can only run one job at a time, if the loader is `busy`, it will return an error on any attempt to create an additional job.
```

Comment:
```suggestion
The loader can only run one job at a time; if the loader is `busy`, it will return an error on any attempt to create an additional job.
```